OxfordBodleian/Juel-Jensen/BDLaethg31.xml

```diff
@@ -228,78 +230,100 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         A stanza has been added in a secondary hand in the lower margin of <locus target="#25v"/> and in the upper margin of <locus target="#26r"/>.
                                     </desc>
                                 </item>
                             </list>
                         </additions>
-                        
+
                         <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1">Two boards made of <material key="paper"/> and covered with dark red <material key="leather"/>.
-                                    The edges are gilded with floral pattern. The red guard leaves (<!-- The German word "Seidenpapier" was used in the pre-filled record [MV] -->) are textured and shining.</decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                                    <material key="paper"/> 
-                                    <material key="leather"/>  
-                                </decoNote>
-                                <decoNote xml:id="b3" type="Cover" color="dark red">
-                                    The front and back covers are provided with elaborate decorations.</decoNote>
-                                <decoNote xml:id="b4" type="SewingStations">5</decoNote>
-                                <decoNote xml:id="b5" type="Spine">The spine is covered with <material key="leather"/>.</decoNote>
+                            <binding xml:id="binding" contemporary="false non-Ethiopian"><!--17th cent. early 18th?-->
+                              <p>
+                                The binding does not possess the characteristics of a traditional Ethiopian binding.
+                                The decorative style, the materials, and the sewing technique used do not match those typically found in Ethiopian bindings.
+                                Additionally, faint traces left by a previous sewing are slightly visible. It is also evident that the bookblock has been re-trimmed
+                                in order to decorate the edges when the new binding was added.
+                              </p>
+                              <decoNote xml:id="b1" type="bindingMaterial">
+                                Binding in <material key="cardboard">cardboard</material> boards covered with dark red blind-tooled <material
+                                key="leather">leather</material>.
+                                The edges are gauffred and gilded with floral pattern. A red, textured, and shining <material key="textile">textile</material>
+                                has been pasted to the inner sides of both boards and to the recto of the first and the verso of the last guard leaves (<!-- The German word "Seidenpapier" was used in the pre-filled record [MV] -->)
```

Comment:
I have added this remark because I do not know whether the word 'Seidenpapier' is a technical term in codicology and could be used. If not, I think this comment can be deleted

Reply:
I could be wrong, but it does not seem to me that the word 'seidenpapier' is a technical term in codicology.

Reply:
Good, then the comment can be removed!